### DISCUSSION TITLE
Password component toggle mask icon

### DISCUSSION THREAD

**Discussion by wrightdean** [primefaces/community#1792] [PrimeReact] [answered]
I have updated my project to use PrimeReact v10.6.3 today and noticed the toggle mask icon for the Password component has moved to the left and looks out of place. Having checked the online docs I can see the same has happened in the given example at [https://primereact.org/password/](https://primereact.org/password).

Is this a bug or does the code need adjusting to take account for some underlying changes, if so the example code needs updating.

![image](https://github.com/primefaces/community/assets/32264332/9b94020e-8c26-4519-990d-9d49d4386c0c)

**Comment by melloware** [comment, marked as answer]
Duplicate of https://github.com/primefaces/primereact/issues/6396
